src/pages/index.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import Header from '../components/header/Header'
-import '../styles/global.css'
+import Layout from '../components/layout/Layout'
+import styles from '../styles/global.css'
```

Comment:
Minor but this is thorwing an eslint error because the `styles` variable isn't being used. Why not just `import '../syles/global.css'`?

or better yet, you could just import the global styles [for the whole app in gatsby-browser.js](https://www.gatsbyjs.org/docs/global-css/#adding-global-styles-without-a-layout-component) rather than for each page individually?

Reply:
Thanks for the feedback! I added the global styles to the gatsby-browser.js file and everything appears to be working.